MyString.java

```diff
@@ -64,10 +99,13 @@ public static String spacedString(String str) {
      * @return a randomly generated string, consisting of 'n' lowercase letters
      */
     public static String randomStringOfLetters(int n) {
-        //// Replace the following statement with your code
-        return null;
+        String abc= "abcdefghijklmnopqrstuvwxyz";
+        String randomString="";
+        for (int i=0; i<n; i++){
+            randomString+= ""+ abc.charAt((int)(Math.random()*26));

```

Comment:
26 is magic number, what does it mean? 
use a constant number of letters of alphabet. or use abc.length..